src/Okta.Sdk.UnitTests/UsersClientShould.cs

```diff
@@ -124,5 +125,464 @@ public async Task RemoveApplicationTargetFromApplicationAdministratorRoleForUser
+                                    ""lastUpdated"": ""2018-03-09T03:18:06.000Z"",
+                                    ""expiresAt"": ""2018-03-16T03:18:06.000Z"",
+                                    ""issuer"": ""https://${yourOktaDomain}/oauth2/ausain6z9zIedDCxB0h7"",
+                                    ""clientId"": ""0oabskvc6442nkvQO0h7"",
+                                    ""userId"": ""00u5t60iloOHN9pBi0h7"",
+                                    ""scopes"": [
+                                      ""offline_access"",
+                                      ""car:drive""
+                                    ],
+                                    ""_links"": {
+                                      ""app"": {
+                                        ""href"": ""https://${yourOktaDomain}/api/v1/apps/0oabskvc6442nkvQO0h7"",
+                                        ""title"": ""Native""
+                                      },
+                                      ""self"": {
+                                        ""href"": ""https://${yourOktaDomain}/api/v1/users/00u5t60iloOHN9pBi0h7/clients/0oabskvc6442nkvQO0h7/tokens/oar579Mcp7OUsNTlo0g3""
+                                      },
+                                      ""revoke"": {
+                                        ""href"": ""https://${yourOktaDomain}/api/v1/users/00u5t60iloOHN9pBi0h7/clients/0oabskvc6442nkvQO0h7/tokens/oar579Mcp7OUsNTlo0g3"",
+                                        ""hints"": {
+                                          ""allow"": [
+                                            ""DELETE""
+                                          ]
+                                        }
+                                      },
+                                      ""client"": {
+                                        ""href"": ""https://${yourOktaDomain}/oauth2/v1/clients/0oabskvc6442nkvQO0h7"",
+                                        ""title"": ""Example Client App""
+                                      },
+                                      ""user"": {
+                                        ""href"": ""https://${yourOktaDomain}/api/v1/users/00upcgi9dyWEOeCwM0g3"",
+                                        ""title"": ""Saml Jackson""
+                                      },
+                                      ""authorizationServer"": {
+                                        ""href"": ""https://${yourOktaDomain}/api/v1/authorizationServers/ausain6z9zIedDCxB0h7"",
+                                        ""title"": ""Example Authorization Server""
+                                      }
+                                    }
+                                  }
+                                ]";
+            var mockRequestExecutor = new MockedStringRequestExecutor(rawResponse);
+            var client = new TestableOktaClient(mockRequestExecutor);
+            await client.Users.ListRefreshTokensForUserAndClient("foo", "bar").ToListAsync();
```

Comment:
Let's check that the response is parsed properly and we get the object we are expecting.